.github/workflows/main.yml

```diff
@@ -23,6 +23,17 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install .[dev]
+          python -m pip install coverage
+          python -m pip install coveralls
       - name: Test with pytest
         run: |
           py.test tests
+          coverage run --branch --include=clvm/SExp.py -m pytest tests/as_python_test.py
+          coverage run -a --branch --include=clvm/serialize.py -m pytest tests/serialize_test.py
+          coverage run -a --branch --source=clvm --omit=clvm/SExp.py,clvm/serialize.py -m pytest tests
```

Comment:
instead of just running all tests and merging all test coverage, I specifically *just* measure test coverage of `SExp.py` based on the deliberate tests written for it, in `as_python_test.py`. Same thing with `serialize.py`. This is to avoid "collateral" coverage by unrelated tests that don't necessarily validate the result of that particular code being executed. I think this gives a more honest representation of what's being tested.